src/models/role.ts

```diff
@@ -0,0 +1,30 @@
+import { DataTypes, Model, Sequelize } from "sequelize";
+import internal from "stream";
+
+export class Role extends Model {
+  declare role_id: number;
+  declare name: string;
+}
+
+export default (sequelize: Sequelize) =>
+  Role.init(
+    {
+      role_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      name: {
+        type: DataTypes.STRING(45),
+        allowNull: false,
+      },
+    },
+    {
+      sequelize,
+      modelName: "Role",

```

Comment:
I believe this line is redundant, as it is the default since that is what the class name is. I will not block pull request over it though.